package.json

```diff
@@ -61,8 +61,6 @@
     "@react-navigation/native": "6.0.6",
     "@react-navigation/stack": "6.0.11",
     "@types/node": "16.9.6",
-    "@credo-ts/core": "patch:@credo-ts/core@npm%3A0.5.11#./.yarn/patches/@credo-ts-core-npm-0.5.11-9122d8c9b8.patch",
-    "@credo-ts/indy-vdr": "patch:@credo-ts/indy-vdr@npm%3A0.5.11#./.yarn/patches/@credo-ts-indy-vdr-npm-0.5.11-ced84362c9.patch",
```

Comment:
These patches are needed still. The changes in them are not in 0.5.13 (or 0.6.x for that matter)

Reply:
@bryce-mcmath I see sonar cloud is fine and passing, not sure where is the "tons"
Will attempt a try to update the patches, will get back if I have any issues with that

Reply:
Sounds good. The link is pretty small but it's in the sonarqubecloud comment, there are 18 new issues apparently. You can't always fix all of them but we should do our best to fix as many as we can